src/wallet/rpcwallet.cpp

```diff
@@ -2563,99 +2570,97 @@ Value zc_raw_pour(const json_spirit::Array& params, bool fHelp)
     if (params[4].get_real() != 0.0)
         vpub_new = AmountFromValue(params[4]);
```

Comment:
When we redesign the RPC interface, let's make sure that amounts are in zatoshi, not ZEC. (The protocol should not even really know about ZEC at all.)